pkg/controller/controller_test.go

```diff
@@ -90,6 +92,26 @@ var _ = Describe("controller.Controller", func() {
 
 			close(done)
 		})
+
+		It("should not leak goroutines when stop", func(done Done) {
+			// TODO(community) - After closing the proper leaks on watch this must be reduced to 0
```

Comment:
should we maybe also document why are those leaks expected and what goroutines are leaking for future reference?

Reply:
yeah.  We can at least link to the relevant issue somewhere around here.

Reply:
(also, for future reference, `community` is not actually a valid TODO target -- this isn't supposed to be who does the TODO, it's supposed to be who has context)